scripts/test-data/data/mostReadGeo.json

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
Probably best to put this under the fixtures folder. The /scripts/ folder currently, more or less, only have scripts in it